### PR TITLE
Bugfix FXIOS-5428 [v112] "Open in external app" prompt is incorrectly displayed

### PR DESCRIPTION
**Pull request by PARAIPAN9**
## Video

https://user-images.githubusercontent.com/51127880/221608098-7cb0b73b-5f17-4960-a336-0de12980ccd4.mp4

### CODE REVIEW

**Comment by dnarcese** [comment]
Can it be dynamic with a max height? Ideally it can be as small as need to fix the text and if it needs more room, it will grow to half the screen size and become scrollable

**Comment by thatswinnie** [comment]
@PARAIPAN9 Looks good on iPhone. On iPad I found an issue in portrait mode when in split view. The buttons are not visible here. Could you please look into this?
Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2023-02-27 at 16 00 13

And if you want to you could remove SnapKit here as well at the same time but it's not a must. We have a task for it otherwise to be picked up at a later time - so feel free to skip that.

**Comment by mte-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Project coverage: 30.13%</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Edited 1 files</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Created 0 files</td>
    </tr>
  </tbody>
</table>

## Client.app: Coverage: 27.03
| File | Coverage ||
| --- | --- | --- |
SnackBar.swift | 0.0% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 649c1661d2b07755bc10accc0f847871c45ecdce
</p>

File: Client/Frontend/Widgets/SnackBar.swift [hunk 1]
```diff
@@ -167,12 +172,9 @@ class SnackBar: UIView {
...
     }
 
-    override func updateConstraints() {
-        super.updateConstraints()
-
+    private func setupButtonViewLayout(constant: CGFloat = 0) {
         buttonsView.snp.remakeConstraints { make in
-            make.top.equalTo(titleView.snp.bottom).offset(UIConstants.DefaultPadding)
-            make.bottom.equalTo(self.snp.bottom)
+            make.bottom.equalTo(self.snp.bottom).inset(constant)
```
Comment (thatswinnie) [inline Client/Frontend/Widgets/SnackBar.swift:177]:
```suggestion
            make.bottom.equalTo(self.safeAreaLayoutGuide.snp.bottom)
```

Comment (thatswinnie) [reply Client/Frontend/Widgets/SnackBar.swift:177]: If we constraint the bottom to the safeAreaLayoutGuide we don't need to set a fixed height inset. It works on every device and so there is no need anymore to check for iPad.

**Pull request merged**